docs/pages/includes/server-access/azure-assign-service-principal.mdx

```diff
@@ -0,0 +1,17 @@
+To grant Teleport permissions, the custom role you created must be assigned to the Teleport service principal -
```

Comment:
Are these partials and images 100% copied from the partial and img under `database-access` subdirs? If so, we should just re-use the existing files and possibly move them to a more general file path.